Packs/Cisco-umbrella/Integrations/CiscoUmbrellaEnforcement/CiscoUmbrellaEnforcement.py

```diff
@@ -0,0 +1,197 @@
+import demistomock as demisto
+from CommonServerPython import *
+
+''' IMPORTS '''
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+
+class Client(BaseClient):
+    """
+    Client to use in the Cisco Umbrella Enforcement integration. Overrides BaseClient
+    """
+
+    def __init__(self, base_url, verify, proxy, api_key):
+        super().__init__(base_url=base_url, verify=verify, proxy=proxy, ok_codes=(200, 202, 204))
+        self.api_key = api_key
+
+    def get_domains_list(self, page: str = '', limit: str = '') -> list:
+        domains_list: list = []
+        response = self.get_domain_request(
+            f'{self._base_url}domains?customerKey={self.api_key}&{prepare_suffix(page=page, limit=limit)}')
+        while response and len(domains_list) < int(limit):
+            response_data = response.get('data', [])
+            for domain in response_data:
+                domain.pop('lastSeenAt')
+                domain['IsDeleted'] = False
+                domains_list.append(domain)
+            response_next_page = response.get('meta', {}).get('next', '')
+            response = self.get_domain_request(response_next_page) if response_next_page else {}
+        return domains_list
+
+    def get_domain_request(self, request: str):
+        return self._http_request('GET', url_suffix='', full_url=request)
+
+    def delete_domains(self, domain_name: str, domain_id: str):
+        res = ''
+        if domain_id:
+            res = self._http_request('DELETE', f"domains/{domain_id}?customerKey={self.api_key}",
+                                     return_empty_response=True)
+        if domain_name:
+            res = self._http_request('DELETE', f"domains?customerKey={self.api_key}&where[name]={domain_name}",
+                                     return_empty_response=True)
+        return res
+
+    def add_event_to_domain(self, event: dict):
+        return self._http_request('POST', f"events?customerKey={self.api_key}", json_data=event,
+                                  return_empty_response=True)
+
+
+def prepare_suffix(page: Optional[str] = '', limit: Optional[str] = '') -> str:
+    """
+    Create the relevant suffix for the domains command,
+     Either there is a complete request that should be sent or page and limit arguments.
+    :param page: The number of the requested page for domains command.
+    :param limit: The limit of the queries to return from the domains command.
+    :return: (str) with the suffix.
+    """
+    suffix = ''
+    if page:
+        suffix += f'page={page}&'
+    if limit:
+        suffix += f'limit={limit}&'
+    return suffix
+
+
+def domains_list_command(client: Client, args: dict) -> CommandResults:
+    """
+    :param client: Cisco Umbrella Client for the api request.
+    :param args: args from the user for the command.
+    """
+    page = args.get('page', '')
+    limit = args.get('limit', '')
```

Comment:
with accordance to the prev int suggestion
```suggestion
    limit = int(args.get('limit', '50'))
```